python/eos/analysis.py

```diff
@@ -455,14 +456,14 @@ def sample_pmc(self, log_proposal, step_N=1000, steps=10, final_N=5000, rng=np.r
         # create PMC sampler
         sampler = pypmc.sampler.importance_sampling.ImportanceSampler(log_target, log_proposal, save_target_values=True, rng=rng)
         generating_components = []
-
-        eps = np.finfo(float).eps
+        proposals = [sampler.proposal]
```

Comment:
Hm. You are not copying the proposal here. Can you confirm that subsequent updates do not modify the old proposals?
If yes, please add a comment that this has been explicitly excluded.
If not, please copy the proposal explicitly.